app/code/Magento/Ui/view/base/web/js/form/element/abstract.js

```diff
@@ -300,7 +300,7 @@ define([
             changed = !utils.compare(rules, this.validation).equal;
 
             if (changed) {
-                this.required(!!rules['required-entry']);
+                this.required(!!this.validation['required-entry'])
```

Comment:
There's a missing `;` at the end here, right?

Reply:
Actually it's optional but to be consistent with the code style I've just added it

Reply:
Optional sure, maybe. But if it's missing, it might screw up the javascript minifier if that one puts all lines on a single line.